test/pkg/VendingMachineTest.java

```diff
@@ -0,0 +1,41 @@
+package pkg;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class VendingMachineTest {
+
+    @Test
+    // getterのテスト 最初の投入金額0円を取得する
+    public void testGetDeposit() {
+        VendingMachine vm = new VendingMachine();
+        int getterTestResult = vm.getDeposit();
+        assertThat(getterTestResult, is(0));
+    }
+
+    @Test
+    // お金投入のテスト 希望金額1000円で上限チェックがOKのとき、trueを返す
+    // 投入金額が0円から1000円に増える
+    public void testInsertMoney1() {
+        VendingMachine vm = new VendingMachine();
+
+        int getterTestResult = vm.getDeposit();
+        assertThat(getterTestResult, is(0));
+
+        boolean insertResult = vm.insertMoney(1000);
+        getterTestResult = vm.getDeposit();
+        assertThat(insertResult, is(true));
+        assertThat(getterTestResult, is(1000));
```

Comment:
```suggestion
        assertThat(getterTestResult, is(inputMoney));
```
投入した金額と同じ、ということをテストしたいなら、変数宣言してコード上で同じものであることを保証した方がいいです。(これぐらいなら間違えないですが、1000000000000000 とかを手打ちした場合、0が少なくても気づかない + テストが実装がバグっていてテストが通っちゃったらまずいので)